src/components/CallView/shared/EmptyCallView.vue

```diff
@@ -114,7 +122,11 @@ export default {
 			if (this.isConnecting) {
 				return t('spreed', 'Connecting …')
 			}
-			return t('spreed', 'Waiting for others to join the call …')
+			let callee = 'others'
+			if (this.isOneToOneConversation()) {
+				callee = this.getDisplayName()
```

Comment:
```suggestion
				return t('spreed', 'Waiting for {user} to join the call', { user: this.getDisplayName() })
```

Reply:
See https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html#javascript for how translations work in Nextcloud